static/app/views/replays/list/replaysList.tsx

```diff
@@ -100,7 +100,7 @@ function ReplaysList() {
               {t('Unindexed search field')}
               <EmptyStateSubheading>
                 {tct('Field [field] requires an [sdkPrompt]', {
-                  field: <strong>'click'</strong>,
+                  field: <strong>&apos;click&apos;</strong>,
```

Comment:
As the file above, I think this example is where the lint rule really helps out. Before the `"` was hard to see inside all the other syntax and special chars... but now that we write `&quot;` it's easier to spot when reading the code.